drivers/prune_tapdisk_logs

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+find /var/log/blktap/ -type f -mtime +30 -delete
```

Comment:
It doesn't really interact with the logrotate. As stated in the commit message the logrotate is very imperfect as the logs have process ids in their names. Any files still open in active processes will be ignored by this anyway as the -HUP reopens the files and thus moves the last mod date on. 

Pkill probably is better but improving/fixing the other script is a separate task if we deem it necessary. Tim has an aversion to `killall` because on some systems it means just that, i.e. kill process 1.